plugin.video.filmarkivet/default.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
```

Comment:
it is usually a good practice to keep this file as smaller as possible since it is the only file that is not compiled to bytecode.
Normally you'd do:

```python
from resources.lib import plugin

plugin.run()
```

Reply:
keep it in mind for a future update

Reply:
Sounds reasonable. I will fix that in next update.